cli/dcoscli/package/main.py

```diff
@@ -682,10 +682,20 @@ def _bundle_uris(uris_directory, zip_file):
     :rtype: None
     """
 
-    for filename in sorted(os.listdir(uris_directory)):
+    uris = sorted(os.listdir(uris_directory))
+    # these uris will be found through a mustache template from the property
+    # name so make sure each name is unique.
+    uri_names = set("-".join(name.split('.')) for name in uris)
```

Comment:
Hmm. This is confusing to read. Using the replace operation is easier to read, no? `name.replace('.', '-')`